src/interface/emacs/khoj.el

```diff
@@ -231,6 +231,11 @@ for example), set this to the full interpreter path."
   :type 'string
   :group 'khoj)
 
+(defcustom khoj-chat-offline nil
```

Comment:
Can you set a boolean for this field to `False` by default?

Reply:
Yes, when `khoj-chat-offline` is `nil`, we set `enable-offline-chat` to `false` in `khoj.yml` 

**Background**:
- While elisp has a bi-state boolean (i.e nil in elisp equates to false). In the khoj config we support a 3 states even for booleans (null, true and false)
- The elisp to json convertors therefore use :false, :null to differentiate between the two possible meanings of nil in a tri-state scenario